src/runtime/EnvironmentRecord.h

```diff
@@ -572,6 +578,8 @@ class DeclarativeEnvironmentRecordNotIndexed : public DeclarativeEnvironmentReco
 
 private:
     bool m_isVarDeclarationTarget : 1;
+    bool m_isCatchClause : 1;
+    Optional<InterpretedCodeBlock::BlockInfo*> m_blockInfo;
```

Comment:
Is `m_blockInfo` necessary? Maybe I'm wrong but it seems that this member variable is nowhere used now.

Reply:
That is correct!